### DISCUSSION TITLE
How do I change the emotions of the speech using the python api?

### DISCUSSION THREAD

**Discussion by Alex-web0** [coqui-ai/TTS#2247] [General Q&A]
I'm new to Coqui TTS and machine learning in general, is there any way to configure the emotions of the output speech? I'm using the latest version of the TTS python library.

**Comment by p0p4k** [comment]
If you are a complete beginner, read the docs first. Read some basics of AI if you can. That will help you answer your own question in a concrete manner.
Good luck!

**Comment by erogol** [comment]
Currently we do not have emotion controls in any of our 🐸TTS models

**Comment by codingaslu** [comment]
Does it have change emotion now ?
